### PR TITLE
docs: Release notes for 0.13.5.

### PR DESCRIPTION
**Pull request by determined-dsw**
## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: docs/release-notes.txt [hunk 1]
```diff
@@ -8,6 +8,57 @@
...
+
+- API: Deprecate Native API. Removed related examples and docs.
+
+- Deployment: Add support for ``det-deploy aws`` in the following new regions: ``ap-northeast-1``, ``eu-central-1``, ``eu-west-1``, ``us-east-2``.
+
+- Documentation: Add extra documentation to and reorganize examples by use case.
+
+- Documentation: Add a ``tf.layers-in-Estimator`` example.
+
+- Kubernetes: Add support for users to specify ``initContainers`` and ``containers`` as part of their custom pod specs. Please see :ref:`custom-pod-specs` for details.
```
Comment (aaron276h) [inline docs/release-notes.txt:34]: Can we also add in that we published `v0.2.0` of the Determined Helm chart

File: docs/release-notes.txt [hunk 2]
```diff
@@ -8,6 +8,57 @@
...
+
+- Trials: Remove support for ``TensorpackTrial``.
+
+- WebUI: Before launching a new TensorBoard through the UI, direct the user to an already running TensorBoard if one exists.
+
+- WebUI: Have experiment and trial detail pages wait for any previous API request polling to finish, before making another request.
+
```
Comment (aaron276h) [inline docs/release-notes.txt:41]: Should we add in that we have published new docker task containers changing TF from `1.15.0` -> `1.15.4` and `2.2.0` -> `2.2.1`

```
determinedai/environments:py-3.6.9-pytorch-1.4-tf-1.15-cpu-0.6.0
determinedai/environments:cuda-10.0-pytorch-1.4-tf-1.15-gpu-0.6.0
determinedai/environments:py-3.6.9-pytorch-1.4-tf-2.2-cpu-0.6.0
determinedai/environments:cuda-10.1-pytorch-1.4-tf-2.2-gpu-0.6.0
```

File: docs/release-notes.txt (hunk 1 above)
Comment (determined-dsw) [reply docs/release-notes.txt:36]: Added.

File: docs/release-notes.txt (hunk 2 above)
Comment (determined-dsw) [reply docs/release-notes.txt:45]: Added.

**Review by aaron276h — APPROVED** [review]
LGTM

**Review by justin-determined-ai — APPROVED** [review]
Overall LGTM, some thoughts below

File: docs/release-notes.txt [hunk 3]
```diff
@@ -8,6 +8,61 @@
...
+- Security: Use one TCP port for all incoming connections to the master and use TLS for all connections if configured.
+
+  - **Breaking Change:** The ``http_port`` and ``https_port`` options in the master configuration have been replaced by the single ``port`` option. The ``security.http`` option is no longer accepted; the master can no longer be configured to listen over HTTP and HTTPS simultaneously.
+
+- Security: Support configuring TLS encryption when deploying Determined on Kubernetes. For more details please see :ref:`install-on-kubernetes`.
+
+- Agent: Increase default max agent starting and idle timeouts to 20 minutes and increase max disconnected period from 5 to 10 minutes.
+
+- API: Deprecate Native API. Removed related examples and docs.
```
[the remark this answers is not in the document]
Comment (determined-dsw) [reply docs/release-notes.txt:26]: I'm not sure how else to word this though.  Any suggestions?

File: docs/release-notes.txt [hunk 4]
```diff
@@ -8,6 +8,61 @@
...
+
+- Documentation: Add a ``tf.layers-in-Estimator`` example.
+
+- Kubernetes: Add support for users to specify ``initContainers`` and ``containers`` as part of their custom pod specs. Please see :ref:`custom-pod-specs` for details.
+
+- Kubernetes: Publish version 0.2.0 of the Determined Helm chart.
+
+- Trials: Remove support for ``TensorpackTrial``.
+
+- WebUI: Before launching a new TensorBoard through the UI, direct the user to an already running TensorBoard if one exists.
```
[the remark this answers is not in the document]
Comment (determined-dsw) [reply docs/release-notes.txt:42]: Ah you are right.  Should we be deleting the proto-release-notes inside docs/release-notes after each release, as I pulled this content from there?

[the remark this answers is not in the document]
Comment (justin-determined-ai) [reply docs/release-notes.txt:42]: Yes, I thought I deleted it. My mistake.
